docs: add static ips for langsmith endpoints for whitelisting (#281)
langchain-ai/docs: src/langsmith/self-host-egress.mdx

In the future, we will be introducing support diagnostics to help us ensure that the LangSmith platform is running at an optimal level within your environment.

<Warning>

>   **This will require egress to `https://beacon.langchain.com` from your network.** Refer to our [whitelisting IP section](/langsmith/cloud-architecture-and-scalability) for static IP addresses, if needed.

Reviewer: ```suggestion
  **This will require egress to `https://beacon.langchain.com` from your network. ** Refer to the [allowlisting IP section](/langsmith/cloud-architecture-and-scalability#allowlisting-ip-addresses) for static IP addresses, if needed.
```
